packages/devtools_app/lib/src/shared/primitives/utils.dart

```diff
@@ -1094,6 +1094,8 @@ class DevToolsFile<T> {
 
 final _lowercaseLookup = <String, String>{};
 
+bool nullOrEmpty(String? str) => str == null || str.isEmpty;
```

Comment:
nit: maybe make this an extension method on `String?`

Reply:
done. Addressed in https://github.com/flutter/devtools/pull/5677